src/expression.d

```diff
@@ -11811,17 +11811,35 @@ public:
  */
 extern (C++) final class CommaExp : BinExp
 {
+    /// This is needed because AssignExp rewrites CommaExp, hence it needs
+    /// to trigger the deprecation.
+    const bool isGenerated;
```

Comment:
This isn't actually private in D.  I'd suggest just leaving it public, we're not systematically using member protection yet in D.